doc/src/manual/noteworthy-differences.md

```diff
@@ -308,3 +308,25 @@ For users coming to Julia from R, these are some noteworthy differences:
     in order to have dynamic dispatch. On the other hand, in Julia every method is "virtual" (although
     it's more general than that since methods are dispatched on every argument type, not only `this`,
     using the most-specific-declaration rule).
+
+## Noteworthy differences from Common Lisp
+
+- Julia uses 1-based indexing for arrays by default, and it can also handle arbitrary [index offsets](@ref man-custom-indices).
+
+- Functions and variables share the same namespace (“Lisp-1”).
+
+- There is a [`Pair`](@ref) type, but it is not meant to be used as a `COMMON-LISP:CONS`. Various iterable collections can be used interchangeably in most parts of the language (eg splatting, tuples, etc). `Tuple`s are the closest to Common Lisp lists for short collections of heterogeneous elements. Use `NamedTuple`s in place of alists.
+
+- The typical Julia workflow for prototyping also uses continuous manipulation of the image, implemented with the [Revise.jl](https://github.com/timholy/Revise.jl) package.
+
+- Bignums are supported, but conversion is not automatic, ordinary integers [overflow](@ref faq-integer-arithmetic).
```

Comment:
As suggested [previously](https://github.com/JuliaLang/julia/pull/16927#discussion_r77824498), I'd go with `;` after "automatic" to avoid a [comma splice](https://en.wikipedia.org/wiki/Comma_splice).

Reply:
Thanks for the grammar corrections. Sorry that I overlooked them when porting this, and I appreciate that you suggested them again.

Reply:
No problem :) it doesn't cost me much, and it makes the final text more readable IMO, so I was happy to do it (both times).